config/templates/hashicorp_vault/lua/audit.lua

```diff
@@ -245,6 +252,29 @@ function path_to_service_target(tag, timestamp, record)
                 code = 2
             end
         end
+        if string.sub(path, 0, 7) == "groups/" then
+            path_segment = parse_string(path,"[^/]+")
+            local groupname = path_segment[3]
+            local service = path_segment[4]            
+            if groupname ~= nil then
+                record["source.user.group.name"] = groupname
+                code = 2
+            end
+            if service ~= nil then
```

Comment:
I don't think there's any convention (or enforcement) that the path segments under groups have any particular meaning.

Reply:
Remove group name